src/order.ts

```diff
@@ -1,643 +1,230 @@
+export type OrderSession = 'NORMAL' | 'AM' | 'PM' | 'SEAMLESS';
+export type OrderDuration = 'DAY' | 'GOOD_TILL_CANCEL' | 'FILL_OR_KILL';
+export type OrderType =
+  | 'MARKET'
+  | 'LIMIT'
+  | 'STOP'
+  | 'STOP_LIMIT'
+  | 'TRAILING_STOP'
+  | 'MARKET_ON_CLOSE'
+  | 'EXERCISE'
+  | 'TRAILING_STOP_LIMIT'
+  | 'NET_DEBIT'
+  | 'NET_CREDIT'
+  | 'NET_ZERO';
+
+export type CancelTime = { date?: string; shortFormat?: boolean };
+
+export type ComplexOrdertStrategyType =
```

Comment:
Thanks for the changes @jdgriffith I really appreciate it. The only thing here is removing the t between Order and Strategy